src/pal/src/safecrt/internal_securecrt.h

```diff
@@ -23,6 +23,13 @@
 #ifndef _INC_INTERNAL_SECURECRT
 #define _INC_INTERNAL_SECURECRT
 
+#ifndef __GLIBC__    // e.g. OS with musl-libc
+typedef unsigned long __uint64_t;
+typedef signed long __int64_t;
```

Comment:
I have removed the commit https://github.com/dotnet/coreclr/commit/3693f3ab03b8df70800e5313ff784b27682eda4a because it was failing the build on other linux (CentOS, Ubuntu, Gentoo):

```
[  2%] Building C object src/pal/src/CMakeFiles/coreclrpal.dir/safecrt/safecrt_input_s.c.o
In file included from /root/coreclrsrc/pal/src/safecrt/safecrt_input_s.c:45:
/root/coreclrsrc/pal/src/safecrt/input.inl:261:13: error: expected ';' after
      expression
    uint64_t num64 = 0LL;             /* temp for 64-bit integers          */
            ^
            ;
/root/coreclrsrc/pal/src/safecrt/input.inl:261:5: error: use of undeclared
      identifier 'uint64_t'
    uint64_t num64 = 0LL;             /* temp for 64-bit integers          */
    ^
/root/coreclrsrc/pal/src/safecrt/input.inl:261:14: error: use of undeclared
      identifier 'num64'
    uint64_t num64 = 0LL;             /* temp for 64-bit integers          */
             ^
```

Can this cleanup be done separately? That way this patch would continue to add only four typedefs in order to successfully build the product on Alpine Linux and harmless to other currently supported distros.